platform/pulp/platform/models/content.py

```diff
@@ -112,6 +115,7 @@ class Artifact(Model):
 
     file = models.FileField(db_index=True, upload_to=StoragePath(), max_length=255)
     downloaded = models.BooleanField(db_index=True, default=False)
+    requested = models.BooleanField(db_index=True, default=False)
```

Comment:
This particular field is primarily useful for the ``deferred_download`` task since it would just query for all artifacts that have ``downloaded = False`` and ``requested = True``, but it could also be helpful if people want to know what content is or isn't used. It could also be a number and we could track the number of times a piece of content is requested, but that adds the overhead of a database write for each request.